.env.example

```diff
@@ -4,8 +4,6 @@ APP_KEY=n96M1TPG821EdN4mMIjnGKxGytx9W2UJ
 NODE_ENV=development
 CACHE_VIEWS=false
 SESSION_DRIVER=cookie
+SESSION_DOMAIN=example.com
+SESSION_COOKIE=example-session
```

Comment:
the reason behind `SESSION_COOKIE & SESSION_DOMAIN` is to minimize being effected by any bug that could be used by an attacker.